### ISSUE TITLE
vm_deduplicated_samples_total should only be yielded when relevant

### ISSUE THREAD

**Issue by hekmon** [VictoriaMetrics/VictoriaMetrics#345]
**Describe the bug**
The `vm_deduplicated_samples_total` metric is yielded by all cluster modules:

* insert
  * `vm_deduplicated_samples_total{type="merge"}`
  * `vm_deduplicated_samples_total{type="select"}`
* storage
  * `vm_deduplicated_samples_total{type="merge"}`
  * `vm_deduplicated_samples_total{type="select"}`
* select
  * `vm_deduplicated_samples_total{type="merge"}`
  * `vm_deduplicated_samples_total{type="select"}`

**Expected behavior**
Each module should only yield the metric if relevant:

* insert
  * n/a
* storage
  * `vm_deduplicated_samples_total{type="merge"}`
* select
  * `vm_deduplicated_samples_total{type="select"}`

**Version**
```
vminsert-20200226-204015-tags-v1.34.1-cluster-0-g52fe4e68
vmstorage-20200226-204024-tags-v1.34.1-cluster-0-g52fe4e68
vmselect-20200226-204020-tags-v1.34.1-cluster-0-g52fe4e68
```

**Comment by valyala** [comment]
This should be fixed in cf9aee4ec39aace2a903fd33464edc8bf0745821 .

**Comment by valyala** [comment]
@hekmon , the bugfix has been included in [v1.34.2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
